katello/katello-restore

```diff
@@ -44,7 +44,9 @@ def restore_databases
     `tar --selinux --overwrite --listed-incremental=/dev/null -xzf pgsql_data.tar.gz -C /`
   else
     `service postgresql start`
+    `runuser - postgres -c "createuser foreman"`
```

Comment:
I don't think many people have actually done this but you can change the db_username using foreman-installer. It may be worth parsing `/etc/foreman-installer/scenarios.d/katello-answers.yaml` for the answer and defaulting to `foreman`?

@jlsherrill what do you think? Its also true of the line below for db name. Maybe others as well.